content/influxdb/cloud-serverless/admin/tokens/update-tokens.md

```diff
@@ -0,0 +1,147 @@
+
+### Enable or disable a token in the InfluxDB UI
+
+1.  On the [token management page](#update-a-token-in-the-influxdb-ui), find the
+    token that you would like to enable or disable.
+2.  Click the token description.
+3.  Click the **{{< icon "toggle-blue" >}} Active** toggle.
+
+{{% /tab-content %}}
+<!----------------------------------- END UI ---------------------------------->
+
+<!--------------------------------- BEGIN CLI --------------------------------->
+{{% tab-content %}}
+
+## Enable a token using the influx CLI
+
+Use the [`influx auth active` command](/influxdb/cloud-serverless/reference/cli/influx/auth/active)
+to activate a token.
+
+Provide the following flags:
+
+- `--token`: API token with permission to update authorizations
+- `--id`: Authorization ID to enable (available in the output of
+  [`influx auth list`](/influxdb/cloud-serverless/reference/cli/influx/auth/list))
+
+{{% code-placeholders "(API|AUTHORIZATION)_(TOKEN|ID)" %}}
+```sh
+influx auth active \
+  --token API_TOKEN \
+  --id AUTHORIZATION_ID
+```
+{{% /code-placeholders %}}
+
+### Disable a token using the influx CLI
+
+Use the [`influx auth inactive` command](/influxdb/cloud-serverless/reference/cli/influx/auth/active)
+to deactivate a token.
+
+Provide the following flags:
+
+- `--token`: API token with permission to update authorizations
+- `--id`: Authorization ID to disable (available in the output of
+  [`influx auth list`](/influxdb/cloud-serverless/reference/cli/influx/auth/list))
+
+{{% code-placeholders "(API|AUTHORIZATION)_(TOKEN|ID)" %}}
+```sh
+influx auth inactive \
+  --token API_TOKEN \
+  --id AUTHORIZATION_ID
+```
+{{% /code-placeholders %}}
+
+{{% /tab-content %}}
+<!---------------------------------- END CLI ---------------------------------->
+
+<!--------------------------------- BEGIN API --------------------------------->
+{{% tab-content %}}
+
+## Update a token using the InfluxDB API
+
+Use the `/api/v2/authorizations` InfluxDB API endpoint to update the description
+and status of a token.
+
+{{< api-endpoint method="PATCH" endpoint="https://cloud2.influxdata.com/api/v2/authorizations/{AUTH_ID}" api-ref="/influxdb/v2.7/api/#operation/PatchAuthorizationsID" >}}
+
+Include the following in your request:
+
+- **Headers**:
+  - **Authorization**: `Bearer API_TOKEN`
+    (API token with the [`write: authorizations`](/influxdb/cloud-serverless/api/#operation/PostAuthorizations) permission)
+  - **Content-type**: `application/json`
+- **Path parameters**:
+  - **authID**: Authorization ID to update
+- **Request body**: JSON object with
+  [authorization properties](/influxdb/cloud-serverless/admin/tokens/create-token/?t=InfluxDB+API#create-a-token-using-the-influxdb-api)
+  to update
+
+### Disable a token
+
+{{% code-placeholders "API_TOKEN" %}}
+```sh
+# Update the description and status of the first authorization listed for the user.
+
+curl --request GET \
+  "https://cloud2.influxdata.com/api/v2/authorizations?user=user2" \
```

Comment:
Are `"` necessary if we're not quoting below?

Reply:
No, they're not. I'll take them out (copypasta)